server/internal/app/repo.go

```diff
@@ -75,11 +75,19 @@ func initReposAndGateways(ctx context.Context, conf *Config, debug bool) (*repo.
 		conf.Task.GCSBucket = conf.GCS.BucketName
 		taskRunner, err := gcp.NewTaskRunner(ctx, &conf.Task)
 		if err != nil {
-			log.Fatalln(fmt.Sprintf("task runner: init error: %+v", err))
+			log.Fatalln(fmt.Sprintf("task runner: gcp init error: %+v", err))
 		}
 		gateways.TaskRunner = taskRunner
+		log.Info("task runner: GCP is used")
```

Comment:
```suggestion
		log.Infofc(ctx, "task runner: GCP is used")
```

Use `infof` even if no variables